### PR TITLE
API realization

### PR DESCRIPTION
**Pull request by derlaft**
Seems to be ready

### CODE REVIEW

File: mexbt.go [hunk 1]
```diff
@@ -0,0 +1,478 @@
...
+	signedRequest
+	Side      string  `json:"side"`
+	OrderType int     `json:"orderType"`
+	Qty       float64 `json:"qty,string"`
+	Px        float64 `json:"px,string"`
+}
+
+type withdrawRequest struct {
+	signedRequest
+	Amount        float64 `json:"amount"`
```
Comment (williamcoates) [inline mexbt.go]: Unfortunately the amount has to be a string for this call. This can be only tested in production currently :(

**Comment by derlaft** [comment]
Yes, it is strange that withdraw returns empty string in sandbox mode. I did not write a test for it because of it

**Comment by williamcoates** [comment]
Could you also implement a convenience function to get deposit address by currency, or alter the response of the current function to return some kind of hash so you can easily grab e.g the 'btc' deposit address without needing to loop through. Then I think we are done :)

**Comment by derlaft** [comment]
Oh wait a minute

**Comment by derlaft** [comment]
Done

**Pull request merged**